README.md

```diff
@@ -499,11 +280,24 @@ absolute path as on the host.
 By default, Docker container restricts the access to `perf_event_open`
 syscall. There are 3 alternatives to allow profiling in a container:
 1. You can modify the [seccomp profile](https://docs.docker.com/engine/security/seccomp/)
-or disable it altogether with `--security-opt seccomp=unconfined` option. In
-addition, `--cap-add SYS_ADMIN` may be required.
+   or disable it altogether with `--security-opt seccomp=unconfined` option. In
+   addition, `--cap-add SYS_ADMIN` may be required.
 2. You can use "fdtransfer": see the help for `--fdtransfer`.
 3. Last, you may fall back to `-e ctimer` profiling mode, see [Troubleshooting](#troubleshooting).
 
+## Building
+
+Build status: [![Build Status](https://github.com/async-profiler/async-profiler/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/async-profiler/async-profiler/actions/workflows/ci.yml)
+
+### Build requirements
+* JDK: 11+
+* GCC: 4.7+
+* C++ Standard: 17-
```

Comment:
17+

Reply:
Be it `+` or `-`, C++ standard cannot be a _build_ requirement. It simply does not make sense.
What matters is the compiler version.
Our docker image has gcc 8.3.1. Ubuntu 18.04, one of the oldest distributions to keep support for, has gcc 7.5.0.
I think we can claim 7.5.0 as the minimal supported GCC version. It has full support for C++14.

Reply:
Thanks Andrei. I will make the change